README.md

```diff
@@ -223,13 +223,13 @@ export type UserFriendDocument = mongoose.Types.Subdocument & {
   uid: UserDocument["_id"] | UserDocument;
 } & UserFriend;
 
-export type UserDocument = mongoose.Document & {
-  metadata?: any;
-  friends: mongoose.Types.DocumentArray<UserFriendDocument>;
-  city: {};
-  name: any;
-  isMetadataString: (this: any) => boolean;
-} & User;
+export type UserDocument = mongoose.Document &
+  UserMethods & {
+    metadata?: any;
+    friends: mongoose.Types.DocumentArray<UserFriendDocument>;
+    city: {};
+    name: any;
```

Comment:
Why is this `any`?

Reply:
For `metadata` see https://github.com/Bounced-Inc/mongoose-tsgen/pull/18#discussion_r542964445. `name: any` is because `name` is declared as a virtual property. These don't have defined types so cannot be pulled from the Mongoose schemas. We could try to grab these with ts-morph in the future.